src/handle.js

```diff
@@ -21,18 +22,27 @@ const prettyKey = (key) => {
 }
 
 const handlers = {
-  init ([name], flags) {
+  init ([name, template], flags) {
     if (!name) {
       reporter.fatal('No name specified')
     }
 
-    pkg.write({
-      appName: name,
-      version: '1.0.0',
-      roles: [],
-      path: 'src/App.sol'
-    }).then(({ appName }) => {
-      reporter.info(`Created new module ${appName}`)
+    if (!template) {
+      template = 'react'
+    }
+
+    // Determine template repository name
+    const aliasedTemplates = {
+      bare: 'aragon/aragon-bare-boilerplate',
+      react: 'aragon/aragon-react-boilerplate'
+    }
+    if (aliasedTemplates[template]) template = aliasedTemplates[template]
```

Comment:
We should show an error (or at least warn the user) if an unknown template name is provided, instead of defaulting to the react boilerplate by default

Reply:
Would 8f68ca5 suffice for that?

Reply:
Yeah